src/views/Details/NFTTransactionDetails.vue

```diff
@@ -247,6 +236,11 @@ export default {
     getAssetIcon,
     prettyFiatBalance,
     getItemIcon,
+    async sendFees() {
+      // TODO: this calculation is incorrect!!!
```

Comment:
can you specify that the fee calculation is not correct for the NFT transactions